From: Michael Niedermayer <michael@niedermayer.cc>
Date: Tue, 22 May 2018 18:57:13 +0000 (+0200)
Subject: libavformat/mxfenc: Add some () to attempt to workaround build issue on osx
X-Git-Url: http://git.ffmpeg.org/gitweb/ffmpeg.git/commitdiff_plain/df3d962d2e736c0748b91a3c8c701d7cb96154a8?hp=1386a199f9fe3d1ab3a6bb8a4e2fe5a9961024e8

libavformat/mxfenc: Add some () to attempt to workaround build issue on osx

fixes ticket7209

Tested-by: Jim DeLaHunt <from.ffmpeg-dev@jdlh.com>
Signed-off-by: Michael Niedermayer <michael@niedermayer.cc>
---

diff --git a/libavformat/mxfenc.c b/libavformat/mxfenc.c
index 44425bf..77f60f5 100644
--- libavformat/mxfenc.c.orig
+++ libavformat/mxfenc.c
@@ -1149,23 +1149,23 @@ static int get_trc(UID ul, enum AVColorTransferCharacteristic trc)
     switch (trc){
     case AVCOL_TRC_GAMMA28   :
     case AVCOL_TRC_GAMMA22   :
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x01,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x01,0x00,0x00}), 16);
         return 0;
     case AVCOL_TRC_BT709     :
     case AVCOL_TRC_SMPTE170M :
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x02,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x02,0x00,0x00}), 16);
         return 0;
     case AVCOL_TRC_SMPTE240M :
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x03,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x01,0x04,0x01,0x01,0x01,0x01,0x03,0x00,0x00}), 16);
         return 0;
     case AVCOL_TRC_BT1361_ECG:
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x06,0x04,0x01,0x01,0x01,0x01,0x05,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x06,0x04,0x01,0x01,0x01,0x01,0x05,0x00,0x00}), 16);
         return 0;
     case AVCOL_TRC_LINEAR    :
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x06,0x04,0x01,0x01,0x01,0x01,0x06,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x06,0x04,0x01,0x01,0x01,0x01,0x06,0x00,0x00}), 16);
         return 0;
     case AVCOL_TRC_SMPTE428  :
-        memcpy(ul, (UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x08,0x04,0x01,0x01,0x01,0x01,0x07,0x00,0x00}, 16);
+        memcpy(ul, ((UID){0x06,0x0E,0x2B,0x34,0x04,0x01,0x01,0x08,0x04,0x01,0x01,0x01,0x01,0x07,0x00,0x00}), 16);
         return 0;
     default:
         return -1;
